journalism_workflow/v1/version.json

```diff
@@ -1,5 +1,5 @@
 {
-  "slug": "journalism_v1",
+  "slug": "v1",
```

Comment:
Calling out that I'm renaming the journalism/simple_workflow version slugs here. This will cause confusion for anyone who has already loaded these workflows into the DB, but I don't think a) any such person exists other than me and b) it's a big issue, since these are demo workflows.